src/components/AppSwitch.js

```diff
@@ -43,6 +44,8 @@ class AppSwitch extends React.Component<LoadingProps, {}> {
       goodWallet.isCitizen(),
       delay(TIMEOUT)
     ])
+    let topWalletRes = API.verifyTopWallet()
```

Comment:
shouldn't this need to be inside the promise all with await to ensure that the user has a proper balance to proceed?
I'm assuming that native token is needed for claim, sell, buy, etc. So I think we should either wait until the daily token is in users wallet or we put a verifyTopWallet() as a first step in send operations.

Reply:
I wanted this to happen in background since txs are 5sec.
Perhaps we can check his balance first and if its near empty then wait.
i'll open a new story to handle this and maybe other edge cases